examples/migration/create_complete_campaign_adwords_api_only.py

```diff
@@ -0,0 +1,244 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example creates a search campaign with the help of AdWords Api.
+
+This code example is the first in a series of code examples that shows how to
+create a Search campaign using the AdWords API, and then migrate it to Google
+Ads API one functionality at a time. See other examples in this directory for
+code examples in various stages of migration.
+
+This code example represents the initial state, where the AdWords API is used
+to create a campaign budget, a Search campaign, ad groups, keywords and expanded
+text ads. None of the functionality has yet been migrated to the Google Ads API.
+"""
+
+
+import datetime
+import urllib.parse
+import uuid
+
+from googleads import adwords
+
+# Number of ads being added/updated in this code example.
+NUMBER_OF_ADS = 5
+# The list of keywords being added in this code example.
+KEYWORDS_TO_ADD = ['mars cruise', 'space hotel']
+
+
+def create_campaign_budget(client):
+    """Creates a new budget and returns the newly created budget ID.
+
+    Args:
+        client: An instance of the google.ads.google_ads.client.GoogleAdsClient
+        class.
+
+    Returns:
+        (str) Budget ID of the newly created budget.
+    """
+    budget_service = client.GetService('BudgetService', version='v201809')
+    budget = {
+        'name' : 'Interplanetary Cruise Budget #{}'.format(uuid.uuid4()),
+        'amount': {
+            'microAmount' : '50000000'
+        },
+        'deliveryMethod' : 'STANDARD'
+    }
+    budget_operations = [{
+        'operator': 'ADD',
+        'operand': budget
+    }]
+    # Add budget.
+    results = budget_service.mutate(budget_operations)
+    created_budget = results['value'][0]
+    print('Budget with ID {} and name {} was created'.format(
+           created_budget['budgetId'], created_budget['name']))
+    return created_budget['budgetId']
+
+
+def create_campaign(client, budget_id):
+    """Creates a new campaign and returns the newly created campaign ID.
+
+    Args:
+        client: A google.ads.google_ads.client.GoogleAdsClient instance.
+        budget_id: (str) Budget ID to be referenced while creating Campaign.
+
+    Returns:
+        (str) Campaign ID of the newly created Campaign.
+    """
+    campaign_service = client.GetService('CampaignService', version='v201809')
+    campaign = {
+        'name': 'Interplanetary Cruise #{}'.format(uuid.uuid4()),
+        'advertisingChannelType': 'SEARCH',
+        # Recommendation: Set the campaign to PAUSED when creating it to stop the
+        # ads from immediately serving. Set to ENABLED once you've added
+        # targeting and the ads are ready to serve.
+        'status': 'PAUSED',
+        'biddingStrategyConfiguration': {
+            'biddingStrategyType': 'MANUAL_CPC',
+        },
+        'startDate': (datetime.datetime.now() +
+        datetime.timedelta(1)).strftime('%Y%m%d'),
```

Comment:
Nit: This line doesn't follow Google's Python style guide. ["No hanging indent in a dictionary"](http://google.github.io/styleguide/pyguide.html#s3.4-indentation)